doc/function_types.md

```diff
@@ -841,6 +841,47 @@ int main()
 
 </details>
 
+
+## Multi-objective functions
+
+A multi-objective function is a set of functions or objectives that are
+simultaneously optimized to reach an optima.
```

Comment:
```suggestion
simultaneously optimized to reach an optimum.
```
I know I am being pedantic---`optimum` is the single form, `optima` is plural.  Or, maybe, this should be `optima` since it's multiple objective optimization?  I guess, in this case, since it's plural, it would be `optima` not `an optima`?  I'm not sure what the intent is, so maybe my comment isn't helpful. :)